lib/ansible/cli/inventory.py

```diff
@@ -193,6 +197,18 @@ def dump(self, stuff):
             import yaml
             from ansible.parsing.yaml.dumper import AnsibleDumper
             results = yaml.dump(stuff, Dumper=AnsibleDumper, default_flow_style=False)
+        elif self.options.toml:
+            try:
+                import toml
+                from ansible.plugins.inventory.toml import AnsibleTomlEncoder, convert_yaml_objects_to_native
```

Comment:
Let's put this into `else:` block to only catch error for `import toml`. I don't like swallowing too broad range of exceptions..

Reply:
I'll have to look at it again.  There was a case originally where `toml` would import but I was purposefully allowing an `ImportError` to bubble up for something else in the the 2nd import.